docs/installation/cloud/overview.md

```diff
@@ -41,13 +41,13 @@ With Docker Machine, you can use the same interface to create cloud hosts with D
 
 To do this, you use the `docker-machine create` command with the driver for the cloud provider, and provider-specific flags for account verification, security credentials, and other configuration details.
 
-[Example: Use Docker Machine to provision cloud hosts](cloud-ex-machine-ocean.md) walks you through the steps to set up Docker Machine and provision a Dockerized host on [Digital Ocean](https://www.digitalocean.com/).
+[Example: Use Docker Machine to provision cloud hosts](cloud-ex-machine-ocean.md) walks you through the steps to set up Docker Machine and provision a Dockerized host on <a href="https://www.digitalocean.com/" target="_blank">Digital Ocean</a>).
 
 ## Where to go next
 * [Example: Manual install on a cloud provider](cloud-ex-aws.md) (AWS EC2)
 
 * [Example: Use Docker Machine to provision cloud hosts](cloud-ex-machine-ocean.md) (Digital Ocean)
 
-* [Using Docker Machine with a cloud provider](https://docs.docker.com/machine/get-started-cloud/)
+* For supported platforms, see [Install Docker Engine](../../installation/).
```

Comment:
You should be able to do `../../install.md` the advantage being the links will work in the GitHub repo. for this one and the one below.